cg/meta/demultiplex/demux_post_processing.py

```diff
@@ -172,6 +164,13 @@ def get_all_demultiplexed_flow_cell_dirs(self) -> List[Path]:
                 demultiplex_flow_cells.append(flow_cell_dir)
         return demultiplex_flow_cells
 
+    def get_sample_sheet_path(self, flow_cell_dir_name: str) -> Path:
+        flow_cell_id: str = get_flow_cell_id(flow_cell_dir_name)
+        return get_sample_sheet_path_hk(flow_cell_id=flow_cell_id, hk_api=self.hk_api)
+
+    def get_flow_cell_path(self, flow_cell_dir_name: str) -> Path:
```

Comment:
```suggestion
    def get_demultiplexed_runs_flow_cell_path(self, flow_cell_dir_name: str) -> Path:
```

As to not confuse it with the `flow_cells` i.e. flow_cell_runs directory